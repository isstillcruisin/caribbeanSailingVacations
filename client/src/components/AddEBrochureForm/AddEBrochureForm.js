import React from 'react'

const AddEBrochureForm = props => (
  <form>
    <div className="form-group add-boat-form">
      <label htmlFor="boat-name">
        <strong>EBrochure Name</strong>
      </label>
      <input
        className="form-control"
        id="boat-name"
        type="text"
        value={props.name}
        placeholder="name of the eBrochure..."
        name="name"
        onChange={props.handleInputChange}
        required
      />
    </div>
    <div className="pull-right">
      <button
        onClick={props.handleFormSubmit}
        type="submit"
        className="btn btn-lg btn-danger"
      >
        Submit
      </button>
    </div>
  </form>
)

export default AddEBrochureForm
