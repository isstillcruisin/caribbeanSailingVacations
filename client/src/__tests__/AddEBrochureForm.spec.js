import React from "react";
import { create } from "react-test-renderer";
import AddEBrochureForm from '../components/AddEBrochureForm';

describe("AddEBrochureForm component", () => {
  test("Matches the snapshot", () => {
    const form = create(<AddEBrochureForm name='test' />);
    expect(form.toJSON()).toMatchSnapshot();
  });
});