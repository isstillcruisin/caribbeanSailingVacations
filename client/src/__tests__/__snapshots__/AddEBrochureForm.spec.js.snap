// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddEBrochureForm component Matches the snapshot 1`] = `
<form>
  <div
    className="form-group add-boat-form"
  >
    <label
      htmlFor="boat-name"
    >
      <strong>
        EBrochure Name
      </strong>
    </label>
    <input
      className="form-control"
      id="boat-name"
      name="name"
      placeholder="name of the eBrochure..."
      required={true}
      type="text"
      value="test"
    />
  </div>
  <div
    className="pull-right"
  >
    <button
      className="btn btn-lg btn-danger"
      type="submit"
    >
      Submit
    </button>
  </div>
</form>
`;
